specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscNode.json

```diff
@@ -0,0 +1,957 @@
+          "description": "Gets or sets a list of dsc node reports."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets or sets the next link."
+        }
+      },
+      "description": "The response model for the list dsc nodes operation."
+    },
+    "DscReportResourceNavigation": {
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "Gets or sets the ID of the resource to navigate to."
+        }
+      },
+      "description": "Navigation for DSC Report Resource."
+    },
+    "DscMetaConfiguration": {
+      "properties": {
+        "configurationModeFrequencyMins": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the ConfigurationModeFrequencyMins value of the meta configuration."
+        },
+        "rebootNodeIfNeeded": {
+          "type": "boolean",
+          "description": "Gets or sets the RebootNodeIfNeeded value of the meta configuration."
+        },
+        "configurationMode": {
+          "type": "string",
+          "description": "Gets or sets the ConfigurationMode value of the meta configuration."
+        },
+        "actionAfterReboot": {
+          "type": "string",
+          "description": "Gets or sets the ActionAfterReboot value of the meta configuration."
+        },
+        "certificateId": {
+          "type": "string",
+          "description": "Gets or sets the CertificateId value of the meta configuration."
+        },
+        "refreshFrequencyMins": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the RefreshFrequencyMins value of the meta configuration."
+        },
+        "allowModuleOverwrite": {
+          "type": "boolean",
+          "description": "Gets or sets the AllowModuleOverwrite value of the meta configuration."
+        }
+      },
+      "description": "Definition of the DSC Meta Configuration."
+    },
+    "DscNodeReport": {
+      "properties": {
+        "endTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the end time of the node report."
+        },
+        "lastModifiedTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the lastModifiedTime of the node report."
+        },
+        "startTime": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Gets or sets the start time of the node report."
+        },
+        "type": {
+          "type": "string",
+          "description": "Gets or sets the type of the node report."
+        },
+        "reportId": {
+          "type": "string",
+          "description": "Gets or sets the id of the node report."
+        },
+        "status": {
+          "type": "string",
+          "description": "Gets or sets the status of the node report."
+        },
+        "refreshMode": {
```

Comment:
>refreshMode [](start = 9, length = 11)

Could this be an enum?

Reply:
We aren't planning to enum this currently as the things being returned here are not finalized yet and are subject to change.